Packages/CommonUI/CommonUI/Classes/View/MessageView/Utils/MessageUtils.swift

```diff
@@ -63,7 +66,13 @@ class MessageUtils {
 		}
 	}
 	
-	static func getOtherRectCorner(index: Int, size: Int) -> UIRectCorner {
+    static func getOtherRectCorner(isQuoteMessage: Bool, isForwardMessage: Bool, index: Int, size: Int) -> UIRectCorner {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/5abb0572a2378570e207060e71d026e6cf8ce71f/Packages/CommonUI/CommonUI/Classes/View/MessageView/Utils/MessageUtils.swift#L69"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>MessageUtils.swift:69</code></td>
    </tr>
  </tbody>
</table>